App/Navigation/AppNavigation/Primary/TabNavigator/Threads/index.ts

```diff
@@ -15,7 +15,7 @@ const nav = createStackNavigator(
     AddThread: AddThreadScreen,
     ViewThread: ThreadsDetail,
     PhotoViewer: ThreadPhotoDetail,
-    SharePhoto: AddCaptionScreen,
```

Comment:
We had duplicate route names. Not sure what the expected behavior is in that case, but just made them unique.

Reply:
cool